subprojects/maven/src/integTest/groovy/org/gradle/api/publish/maven/MavenPublishDependenciesIntegTest.groovy

```diff
@@ -80,6 +80,7 @@ class MavenPublishDependenciesIntegTest extends AbstractMavenPublishIntegTest {
 
             dependencies {
                 api $dependencyNotation
+                api 'group:projectB:1.0'
```

Comment:
:o: why this change?

Reply:
Because otherwise this test triggers the new check: it has no version anywhere

Reply:
Ah I see, this tests with _no version_